woocommerce/woocommerce-gutenberg-products-block
# Update product placeholders with products from design

## What

This PR updates the dummy products to use real product info and images.

It also reduces the size of one of the images, it was huge in px and close to 1 Mb.
Fixes https://github.com/woocommerce/woocommerce-blocks/pull/11997#pullrequestreview-1757905545

## Why

To have real product placeholders instead of dummy empty ones.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. On a JN or WooExpress installation, replace the WooCommerce Blocks plugin with this one: https://wcblocks.wpcomstaging.com/wp-content/uploads/woocommerce-gutenberg-products-block-12007.zip
2. Using the plugin editor comment out [this line](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/src/StoreApi/Routes/V1/AI/Product.php#L83) to avoid replacing the dummy products with AI-generated ones.
3. Go through the CYS process.
4. Make sure all the dummy products were created with their corresponding title, description, image and price.
5. Uncomment the line commented out in step 2, reset the CYS and delete all products on `wp-admin/tools.php?page=woocommerce-admin-test-helper`.
6. Go through the CYS process and make sure the products are AI-generated ones and no dummy products are shown.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog
